shared/src/commonMain/sqldelight/me/saket/press/data/shared/Note.sq

```diff
@@ -7,6 +7,7 @@ CREATE TABLE note (
   content TEXT NOT NULL,
   createdAt TEXT AS DateTime NOT NULL,
   updatedAt TEXT AS DateTime NOT NULL,
+  archivedAt TEXT AS DateTime,
```

Comment:
This is missing a SQL migration. The app will 💥 for users who install this update. https://github.com/cashapp/sqldelight#migrations